app/src/androidTest/java/com/duckduckgo/app/statistics/pixels/PixelNameTest.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.statistics.pixels
+
+import org.junit.Assert.fail
+import org.junit.Test
+
+class PixelNameTest {
+
+    @Test
+    fun verifyNoDuplicatePixelNames() {
+        val existingNames = mutableMapOf<String, Pixel.PixelName>()
```

Comment:
nit: consider using a Set, the add method will return false if it doesn't get added (because it's already there).

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-mutable-set/add.html